### DISCUSSION TITLE
表白成功了！！

### DISCUSSION THREAD

**Discussion by MiG-2233** [share121/inter-knot#191] [其他]
哈喽哈喽，我今天跟女神表白，她居然同意了！我有女朋友辣，给大伙看看她的美照🥰🥰也希望大伙能早日脱单呀！![Image_1722521888696](https://github.com/user-attachments/assets/49b3452f-6ace-4b55-aced-d2d6e1829dee)

**Comment by Timely1013** [comment]
恭喜^ω^

**Comment by NOVAQUQ** [comment]
所以美照呢？！

**Comment by luckey-lang** [comment]
恭喜

**Comment by WORLD-executeME** [comment]
祝99

**Comment by MiG-2233** [comment]
等会，我传的图片丢了，我研究一下咋回事

**Comment by capacitor1** [comment]
> Reply::等会，我传的图片丢了，我研究一下咋回事

见[InterKnotForm@195 - 为什么绳匠们都不好好发图片](https://github.com/share121/inter-knot/discussions/195)
🤔好像图片正在上传的时候会显示成Uploading，此时不能着急地发布帖子。

**Comment by MiG-2233** [comment]
这是我的女朋友啦
![Image_1722521888696](https://github.com/user-attachments/assets/86b19b17-1fc4-4235-9f77-196f2bd1942e)

**Comment by 17732386508** [comment]
我为啥一直显示网页版，怎么进入绳网啊

**Reply by sabrin1145** [reply]
不是都是网页版吗

**Comment by dangerousNH4NO3** [comment]
哇！她真好看

**Comment by soyorin2333** [comment]
6

**Comment by ghugfyfgyf** [comment]
恭喜🎉

**Comment by BiancaAtagina** [comment]
恭喜恭喜，她真好看。

**Comment by a0906390287** [comment]
哇真恭喜呢，那麼現在他是我的了

**Comment by 323157364** [comment]
啊？
